### PR TITLE
hello-kubernetes quickstart should show how to curl the running app

### PR DESCRIPTION
**Pull request by yoDon**
# Description

Adds example of how to curl into a dapr app that is running under kubernetes

## Issue reference

This PR will close https://github.com/dapr/quickstarts/issues/370

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] The quickstart code compiles correctly
* [X] You've tested new builds of the quickstart if you changed quickstart code
* [X] You've updated the quickstart's README if necessary

### CODE REVIEW

**Review by wcs1only — CHANGES_REQUESTED** [review]
Our quickstarts are now self validating. You can annotate your change in the markdown file to have it be executed automatically and have its output automatically validated. This helps us make sure our documents are working correctly with every release.

See: https://github.com/dapr/quickstarts/blame/master/hello-kubernetes/README.md#L129-L141 for an example. For the complete documentation of our validation framework, and the checks that it supports, see: https://github.com/dapr/mechanical-markdown/tree/main/examples

If you don't have the time or inclination to give this a shot, let me know and I'll create a separate issue to add validation to these steps.

File: hello-kubernetes/README.md [hunk 1]
```diff
@@ -151,6 +151,26 @@ Minikube users cannot see the external IP. Instead, you can use `minikube servic
 
 Here you can see that two ports are displayed. Both the ports have been injected when Dapr was enabled for this app. Additionally, in this quickstart the HTTP Port is used for further communication with the Dapr sidecar.
 
+Next submit an order to the app
+
+```bash
+curl --request POST --data "{\"data\": { \"orderId\": \"42\" } }" --header "Content-Type:application/json" http://localhost/neworder
```
[the remark this answers is not in the document]
Comment (yoDon) [reply hello-kubernetes/README.md]: I've applied the changes you suggested (localhost -> $NODE_APP) and added automated tests for the curl examples I added.

File: hello-kubernetes/README.md [hunk 2]
```diff
@@ -151,6 +151,26 @@ Minikube users cannot see the external IP. Instead, you can use `minikube servic
...
+```
+
+Expected output:
+Empty reply from server
+
+Confirm the order was persisted by requesting it from the app
+
+```bash
+curl http://localhost/order
```
[the remark this answers is not in the document]
Comment (yoDon) [reply hello-kubernetes/README.md]: I've applied the changes you suggested (localhost -> $NODE_APP) and added automated tests for the curl examples I added.

**Review by yoDon — COMMENTED** [review]
@wcs1only I've applied your changes and the PR is ready to review (github is still reporting them as unresolved, but it wont let me apply your changes as I've already applied them myself)

File: hello-kubernetes/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (wcs1only) [reply hello-kubernetes/README.md]: Awesome, thanks!

File: hello-kubernetes/README.md [hunk 3]
```diff
@@ -151,6 +151,45 @@ Minikube users cannot see the external IP. Instead, you can use `minikube servic
...
+
+```bash
+curl http://$NODE_APP/order
+```
+
+Expected output:
+```json
+{"orderId":"42"}
+```
+
```
Comment (wcs1only) [inline hello-kubernetes/README.md:192]:
```suggestion
<!-- END_STEP -->

```

The missing END_STEP tag was making our autovalidation skip this step. Our parser *should* have thrown an error for this case, which means we probably have an error in our markdown automation tool: https://github.com/dapr/mechanical-markdown . I'll go ahead and fix this so we can get your PR merged, but I'll see if I can make a test for this in the upstream tool.

Comment (wcs1only) [reply hello-kubernetes/README.md:193]: I've created an issue to track the bug you exposed: https://github.com/dapr/mechanical-markdown/issues/8

Thanks!

**Review by wcs1only — APPROVED** [review]

**Pull request merged**